tad/migrations/versions/b62dbd9468e4_create_status_user_and_task_table.py

```diff
@@ -56,17 +56,6 @@ def upgrade() -> None:
     )
     # ### end Alembic commands ###
 
-    # ### custom commands ###
-    op.bulk_insert(
```

Comment:
Why is this removed?

Reply:
My bad, I will make it a migration to rename the exisitng columns so the translations will work.